src/components/AssetManagementPanel/AssetManagementForm/FormVariants/ActionSelectionForm/ActionSelectionForm.test.tsx

```diff
@@ -0,0 +1,138 @@
+import React from "react";
+import { render, fireEvent, act } from "@testing-library/react";
+import "@testing-library/jest-dom";
+import { ActionSelectionForm } from "./ActionSelectionForm";
+
+describe("None", () => {
+  it("should display the non-editable beneficiary & holder when the app is in None state", () => {
```

Comment:
It doesnt care about the state anymore, can change the title to:

```suggestion
  it("should display the non-editable beneficiary & holder", () => {
```

Reply:
ok, updated the title